src/integrators/sdirk_integrators.jl

```diff
@@ -687,11 +678,14 @@ end
     end
   end
 
+  # TODO: Add extrapolant initial guess
```

Comment:
No, this guess is the smoothed aggregation form FSAL, this should not be extrapolated.